packages/palette/src/elements/Modal/index.tsx

```diff
@@ -0,0 +1,234 @@
+import React, { SFC, useEffect, useRef, useState } from "react"
```

Comment:
why is the implementation in `index.tsx` and the export in `Modal.tsx`? Isn't it the other way around in most cases? Also I'd advocate for not even having an index file, FWIW.

Reply:
I just removed the index file -- but this is the convention we've used for every palette component so on one hand it may be good to be consistent.